kbfsgit/runner.go

```diff
@@ -450,6 +453,17 @@ func (r *runner) processGogitStatus(
 	lastByteCount := 0
 	for update := range statusChan {
 		if update.Stage != currStage {
+			if r.verbosity >= 3 {
+				f, err := os.Create(filepath.Join(
+					os.TempDir(), fmt.Sprintf("memprof.%d.prof", update.Stage)))
```

Comment:
Can you output the location of this file as part of the status dump?

Reply:
By "status dump" you mean the errput data that's being printed as part of the `git` execution to the terminal?  Sure, will do.

Reply:
Yep, thanks.